src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -9,7 +9,7 @@ const alertsQuery = require('./alerts.graphql');
 const eventPage = require('./eventPage.graphql');
 const facilitySidebarQuery = require('./navigation-fragments/facilitySidebar.nav.graphql');
 const icsFileQuery = require('./file-fragments/ics.file.graphql');
-
+const bioPage = require('./bioPage.graphql');
```

Comment:
Can you add this the GetLatestPageById query file as well? It may not be necessary, but I'd like to keep these queries in sync as much as possible.

Reply:
Added to getlastestpagebyid and getpagebyid